.github/workflows/sdk-gorelease.yaml

```diff
@@ -0,0 +1,50 @@
+name: GoRelease
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write
+
+jobs:
+  publish-terraform:
```

Comment:
is there a reason this is a separate workflow? Just means end users will need to setup an additional workflow in their repos and means another file the managed flow needs to manage?